packages/api-gateway/src/drivers/http/routes/usuarios/index.js

```diff
@@ -1,31 +1,33 @@
 const { usuariosAdapter } = require('../../adapters');
+
 const {
+  getUsuariosSchema,
```

Comment:
Rename this to getAllUsuariosSchema, getGroupUsuariosSchema, getListUsuariosSchema or other  is easy to mistype getUsuarioSchema and getUsuariosSchema